bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -812,7 +812,8 @@ public long scanJournal(long journalId, long journalPos, JournalScanner scanner)
                         }
                         isPaddingRecord = true;
                     } else {
-                        throw new IOException("Invalid record found with negative length : " + len);
+                        LOG.error("Invalid record found with negative length: {}", len);
+                        throw new IOException();
```

Comment:
Can you please keep the 'message'? It may be useful

Reply:
Updated it. 
On a side note, I understand the test is not the cleanest approach for doing things, however I took some liberty since its test-only code.